src/com/company/Main.java

```diff
@@ -6,6 +6,7 @@
 
     public static void main(String[] args) {
 
+        //initialization
```

Comment:
Variables must have name length > 1.
Extract duplicated code to method (n % divider == 0)

Reply:
Done